python/ray/serve/_private/http_proxy.py

```diff
@@ -291,6 +295,12 @@ def get_handle(name):
                 "method",
             ),
         )
+        self.processing_latency_tracker = metrics.Histogram(
+            "serve_http_request_latency_ms",
+            description="The latency for http request to be processed.",
+            boundaries=DEFAULT_LATENCY_BUCKET_MS,
+            tag_keys=("endpoint",),
```

Comment:
```suggestion
            tag_keys=("route_prefix",),
```
keep the naming standard across the API